exporters/geneva/CMakeLists.txt

```diff
@@ -4,24 +4,33 @@ cmake_minimum_required(VERSION 3.12)
 # (using add_subdirectory) or if it is the main project
 #
 set(MAIN_PROJECT OFF)
-
 set(MAIN_PROJECT OFF)
 if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
   project(opentelemetry-geneva-metrics)
   set(MAIN_PROJECT ON)
 endif()
 
+if(NOT WIN32)
+  find_package(CURL REQUIRED)
```

Comment:
Why is `find_package(CURL...)` moved to be before `find_package(opentelemetry-cpp...)`?

Reply:
Oh I don't think it was for a reason, it just got moved while excluding it for windows.